test/groovy/TransportRequestUploadFileTest.groovy

```diff
@@ -521,56 +517,33 @@ public class TransportRequestUploadFileTest extends BasePiperTest {
         // this one is used since there is nothing in the signature
         nullScript.commonPipelineEnvironment.setMtarFilePath('/path2')
 
-        ChangeManagement cm = new ChangeManagement(nullScript) {
-            void uploadFileToTransportRequestSOLMAN(
-                                              Map docker,
-                                              String changeId,
-                                              String transportRequestId,
-                                              String applicationId,
-                                              String filePath,
-                                              String endpoint,
-                                              String credentialsId,
-                                              String cmclientOpts) {
-
-                cmUtilReceivedParams.filePath = filePath
+        def calledWithParameters,
+            calledWithStepName,
+            calledWithMetadata,
+            calledWithCredentials
+
+        helper.registerAllowedMethod( 'piperExecuteBin', [Map, String, String, List], {
+            params, stepName, metaData, creds ->
+                calledWithParameters = params
+                calledWithStepName = stepName
+                calledWithMetadata = metaData
+                calledWithCredentials = creds
             }
-        }
-
+        )
+        
```

Comment:
As said before, maybe run a formatter to avoid unnecessary spaces in the change.